tests/functional/test_credentials.py

```diff
@@ -198,7 +198,7 @@ def create_assume_role_response(self, credentials, expiration=None):
             },
             'AssumedRoleUser': {
                 'AssumedRoleId': 'myroleid',
-                'Arn': 'arn:aws:iam::1234567890:user/myuser',
+                'Arn': f'arn:aws:iam::{credentials.account_id}:user/myuser',
```

Comment:
Since the `AssumedRoleUser` object with a valid arn was already in these tests, account ID will never resolve to a non-None value.